filebeat/module/elasticsearch/log/config/log.yml

```diff
@@ -0,0 +1,10 @@
+type: log
+paths:
+{{ range $i, $path := .paths }}
+ - {{$path}}
+{{ end }}
+exclude_files: [".gz$"]
```

Comment:
This needs to exclude the "other" log files that are likely in the same directory, see the exclude_files line used here:

https://github.com/joshuar/beats/blob/702f57baf6011dfd97482e7747743319d5915484/filebeat/module/elasticsearch/log/config/log.yml#L5

Reply:
Thanks, copied over. It would be great if in the future this log file would have a unique pattern like `server-*.log` prefix.